lms/static/sass/multicourse/_courses.scss

```diff
@@ -125,8 +157,214 @@
+    border-bottom: 1px solid $courseware-button-border-color;
+    width: 100%;
+    height: auto;
+    max-height: ($baseline*10);
+
+    ul {
+      @include padding-left(0);
+      margin: 0;
+      list-style: outside none none;
+    }
+
+    li {
+      @include float(left);
+      @include margin(($baseline/2), $baseline, ($baseline/2), 0);
+      padding: ($baseline/2) ($baseline*0.75);
+      position: relative;
+      background: $courseware-button-border-color;
+      width: auto;
+
+      a {
+        @extend %t-strong;
+        color: $gray-d2;
+        text-decoration: none;
+
+        i {
+          color: $gray-l2;
+        }
+      }
+    }
+
+    .clear-filters {
+      @include line-height(29.73);
+      @extend %t-icon5;
+      @extend %t-strong;
+      margin: ($baseline/2) 0;
+      width: auto;
+      text-align: center;
+      color: $m-blue-d1;
+    }
+
+    .flt-right {
+      @include float(right);
+    }
+  }
+
+
+
+  .search-facets{
+    @include rtl() { $layout-direction: "LTR"; }
+    @include ltr() { $layout-direction: "RTL"; }
+    @include box-sizing(border-box);
+    @extend %ui-depth1;
+    position: relative;
+    box-shadow: 1px 2px 5px $black-t0;
+    border-top: 1px solid $black;
+    border-bottom: 2px solid $black;
+    background-color: $white;
+    max-height: ($baseline*100);
+
+    @include fill-parent();
+    margin: ($baseline*2) 0 ($baseline*3.5) 0;
+
+    @include media($bp-medium) {
+      @include span-columns(2);
+      @include omega();
+    }
+
+    @include media($bp-large) {
+      @include span-columns(3);
+      @include omega();
+    }
+
+    @include media($bp-huge) {
+      @include span-columns(3);
+      @include omega();
+    }
+
+    &.phone-menu {
+      padding: 0;
+      overflow: visible;
+      border: medium none;
+    }
+
+    &:before {
+      opacity: 0;
+      position: absolute;
+      top: -3px;
+      right: 0;
+      background-color: $white;
+      padding: ($baseline*2) ($baseline*0.75) 0 ($baseline*0.75);
+      height: 5px;
+      content: "";
+      width: ($baseline*2.5);
+    }
+
+    h2,
+    section {
+      @extend %t-icon5;
+      @extend %t-strong;
+      margin: 0 ($baseline/2) 0 ($baseline/2);
+      border: medium none;
+      padding: ($baseline/2);
+      color: $facet-text-color;
+      font-family: $sans-serif;
+      text-transform: none;
+    }
+
+    h3 {
+      @extend %t-icon6;
+      @extend %t-strong;
+      margin: 0 ($baseline/2) ($baseline/2) ($baseline/2);
+      color: $facet-text-color;
+      font-family: $sans-serif;
+    }
+
+    section {
+      margin: 0;
+      border-top: 1px solid $courseware-button-border-color;
+      padding: ($baseline/2) 0 ($baseline/2) 0;
+    }
+
+    a {
+      @include float(left);
+      @include box-sizing(border-box);
+      opacity: 1;
+      padding: 0 12px;
+      width: 100%;
+      overflow: hidden;
+      text-decoration: none;
+      text-overflow: ellipsis;
+      line-height: 28px;
```

Comment:
there's a `@line-height` mixin that's used. it, however, multiplies the parameter by 1.48, FYI.